docs/date.md

```diff
@@ -1,96 +1,389 @@
+```
 
 ## Properties
 
 ### date.**year** : number
 
-Returns the year this `Date` represents
-
 ### date.**month** : number
 
-Returns the month this `Date` represents
-
 ### date.**day** : number
 
-Returns the day this `Date` represents
+The above read-only properties allow accessing each component of the date individually.
+
+Usage examples:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.year   // => 2006
+date.month  // => 8
+date.day    // => 24
+```
 
 ### date.**dayOfWeek** : number
 
-Returns the day of the week this `Date` represents
+The `dayOfWeek` read-only property gives the weekday number that the date falls on.
+The weekday number is defined as in the ISO 8601 standard: a value between 1 and 7, inclusive, with Monday being 1, and Sunday 7.
+For an overview, see [ISO 8601 on Wikipedia](https://en.wikipedia.org/wiki/ISO_8601#Week_dates).
 
-```js
-const date = new Temporal.Date(2019, 11, 18); // Monday
-date.dayOfWeek; // 1
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN'][date.dayOfWeek - 1]  // => THU
+```
+
+### date.**dayOfYear** : number
+
+The `dayOfYear` read-only property gives the ordinal day of the year that the date falls on.
+This is a value between 1 and 365, or 366 in a leap year.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+// ISO ordinal date
+console.log(date.year, date.dayOfYear);  // 2006 236
 ```
 
 ### date.**weekOfYear** : number
 
-Returns the week in a calendar year
+The `weekOfYear` read-only property gives the ISO week number of the date.
+This is normally a value between 1 and 52, but in a few cases it can be 53 as well.
+ISO week 1 is the week containing the first Thursday of the year.
+For more information on ISO week numbers, see for example the Wikipedia article on [ISO week date](https://en.wikipedia.org/wiki/ISO_week_date).
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+// ISO week date
+console.log(date.year, date.weekOfYear, date.dayOfWeek);  // 2006 34 4
+```
 
 ### date.**daysInMonth** : number
 
-Returns the number of days in the month the `Date` Object represents
+The `daysInMonth` read-only property gives the number of days in the month that the date falls in.
+This is 28, 29, 30, or 31, depending on the month and whether the year is a leap year.
+
+Usage example:
+```javascript
+// Attempt to write some mnemonic poetry
+const monthsByDays = {};
+for (let month = 1; month <= 12; month++) {
+    const date = Temporal.now.date().with({month});
+    monthsByDays[date.daysInMonth] = (monthsByDays[date.daysInMonth] || []).concat(date);
+}
+
+const strings = monthsByDays[30].map(date => date.toLocaleString('en', {month: 'long'}));
+// Shuffle to improve poem as determined empirically
+strings.unshift(strings.pop());
+const last = strings.pop();
+const poem = `Thirty days hath ${strings.join(', ')}, and ${last}`;
```

Comment:
You use `toLocaleString()`, which is nice, but the way to format a list is to use Intl.ListFormat.

Reply:
I did not know about that! It seems to be supported in the latest version of Node, despite MDN saying it isn't (out of date?)